framework/Models/EnsembleModel.py

```diff
@@ -485,14 +499,12 @@ def evaluateSample(self, myInput, samplerType, kwargs):
         a mandatory key is the sampledVars'that contains a dictionary {'name variable':value}
       @ Out, returnValue, dict, This holds the output information of the evaluated sample.
     """
-    kwargsKeys = list(kwargs.keys())
-    kwargsKeys.pop(kwargsKeys.index("jobHandler"))
-    kwargsToKeep = { keepKey: kwargs[keepKey] for keepKey in kwargsKeys}
-    jobHandler = kwargs['jobHandler']
+    kwargsToKeep = { keepKey: kwargs[keepKey] for keepKey in list(kwargs.keys())}
```

Comment:
Is this just `kwargsToKeep == copy.deepcopy(kwargs)`? No change needed, I see how this could be useful in the future if we have keys we don't want to keep.

Reply:
ops...yes.. i will address this in next PR